profiler/src/main/java/com/splunk/opentelemetry/profiler/ProfilingSemanticAttributes.java

```diff
@@ -27,9 +27,6 @@
   // removed
   public static final String SCHEMA_URL = "https://opentelemetry.io/schemas/1.4.0";
 
-  public static final AttributeKey<String> LINKED_SPAN_ID = stringKey("span_id");
-  public static final AttributeKey<String> LINKED_TRACE_ID = stringKey("trace_id");
-
```

Comment:
These aren't attributes, they are first-class fields on the logs otlp proto object.